app/database/models/server/my_team.ts

```diff
@@ -20,6 +20,8 @@ export default class MyTeamModel extends Model implements MyTeamModelInterface {
     static table = MY_TEAM;
 
     static associations: Associations = {
+
+        /** A TEAM can be associated to one MY_TEAM (relationship is 1:1) */
```

Comment:
We don't expect to have a MY_TEAM without the team, right? The way it is written seems to imply that you may have a MY_TEAM without a TEAM.

The other way around might be possible (not sure) (i.e. having a team in the database that we do not belong to), but from the MY_TEAM perspective, the team always exist.

Reply:
That's right.  Changing it to `A TEAM is associated to one MY_TEAM (relationship is 1:1) `